charts/argo-rollouts/Chart.yaml

```
@@ -19,4 +19,4 @@ annotations:
     url: https://argoproj.github.io/argo-helm/pgp_keys.asc
   artifacthub.io/changes: |
     - kind: changed
-      description: Bump argo-rollouts to v1.8.0
+      description: argo-rollouts will re-deploy if changes are made to the argo-rollouts configmap.
```

```suggestion
      description: argo-rollouts will re-deploy if changes are made to the argo-rollouts configmap.

```

Add newline to fix linter (new-line-at-end-of-file)